src/fio/fioEngine.js

```diff
@@ -153,18 +223,29 @@ export class FioEngine extends CurrencyEngine {
       this.walletLocalData.totalBalances[currencyCode] = '0'
     }
 
-    this.fioSDK
-      .getFioBalance()
-      .then(value => {
-        nativeAmount = value.balance
-        nativeAmount = nativeAmount + ''
-        this.updateBalance(currencyCode, nativeAmount)
-      })
-      .catch(e => {
-        console.error('checkAccountInnerLoop error: ' + JSON.stringify(e))
-        nativeAmount = '0'
-        this.updateBalance(currencyCode, nativeAmount)
+    try {
+      const { balance } = await this.multicastServers('getFioBalance')
+      nativeAmount = balance + ''
+      this.updateBalance(currencyCode, nativeAmount)
+    } catch (e) {
+      this.log('checkAccountInnerLoop error: ' + JSON.stringify(e))
+      nativeAmount = '0'
+      this.updateBalance(currencyCode, nativeAmount)
```

Comment:
The updateBalance call can be moved outside of the try/catch

Reply:
ok